link/src/test/java/com/stripe/android/link/ui/verification/VerificationViewModelTest.kt

```diff
@@ -0,0 +1,238 @@
+package com.stripe.android.link.ui.verification
+
+import app.cash.turbine.test
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.core.Logger
+import com.stripe.android.core.strings.resolvableString
+import com.stripe.android.link.LinkScreen
+import com.stripe.android.link.TestFactory
+import com.stripe.android.link.account.FakeLinkAccountManager
+import com.stripe.android.link.account.LinkAccountManager
+import com.stripe.android.link.analytics.FakeLinkEventsReporter
+import com.stripe.android.link.analytics.LinkEventsReporter
+import com.stripe.android.link.model.LinkAccount
+import com.stripe.android.model.ConsumerSession
+import com.stripe.android.testing.FakeLogger
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.test.UnconfinedTestDispatcher
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.test.setMain
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+internal class VerificationViewModelTest {
+    private val dispatcher = UnconfinedTestDispatcher()
+
+    @Before
+    fun setUp() {
+        Dispatchers.setMain(dispatcher)
+    }
+
+    @After
+    fun tearDown() {
+        Dispatchers.resetMain()
+    }
+
+    @Test
+    fun `init sends analytics event`() = runTest(dispatcher) {
```

Comment:
This test name doesn't seem accurate anymore - it looks like this is testing that start verification is called, but doesn't actually test analytics

Reply:
there are several other tests which mention analytics in their names but which don't actually test analytics anymore. Can you update them please?